CHANGELOG.md

```diff
@@ -0,0 +1,129 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
+and this project adheres to [Semantic
+Versioning](http://semver.org/spec/v2.0.0.html) except to the first release.
+
+## [Unreleased]
+
+### Added
+
+- Add Coveralls support
```

Comment:
I'm not sure if it's covered by any standard, but I have seen dropping `add` from **Added** and `fix` from **Fixed** changelog section points. It's more a question of taste (and, to be honest, I haven't got a preferable answer), so I don't request any changes here, just raise a question.

Reply:
> I'm not sure if it's covered by any standard

It does not force removing a verb.
See an example of changelog in https://keepachangelog.com/en/1.0.0/. Some entries contain a verb and some are not. I have updated changelog entries for unreleased version and keep as is for 1.5.

```diff
diff --git a/CHANGELOG.md b/CHANGELOG.md
index 64b0238..e761d23 100644
--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -10,12 +10,12 @@ Versioning](http://semver.org/spec/v2.0.0.html) except to the first release.
 
 ### Added
 
-- Add Coveralls support (#149)
-- Add reusable testing workflow (integration testing with latest Tarantool)
-- Add simple CI based on GitHub actions (#114)
-- Add support UUID type in msgpack (#90)
-- Add go modules support (#91)
-- Add utube handling (#85)
+- Coveralls support (#149)
+- Reusable testing workflow (integration testing with latest Tarantool)
+- Simple CI based on GitHub actions (#114)
+- Support UUID type in msgpack (#90)
+- Go modules support (#91)
+- queue-utube handling (#85)
 
 ### Fixed
 
@@ -23,7 +23,7 @@ Versioning](http://semver.org/spec/v2.0.0.html) except to the first release.
 
 ### Changed
 
-- Handle everything with go test (#115)
+- Handle everything with `go test` (#115)
 - Use plain package instead of module for UUID submodule (#134)
 - Make test case consistent with comments (#105)
 - Reset buffer if its average use size smaller than quater of capacity (#95)
 ```